<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TXID</class>
 <widget class="QMainWindow" name="TXID">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>912</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TXID Fixer</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="main">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="selectDir">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Select Directory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeSel">
        <property name="minimumSize">
         <size>
          <width>92</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Remove Selected</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="fileFixed">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fix_shadow">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string>Fix Shadow</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fix">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string>Fix</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QProgressBar" name="progress">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTabWidget" name="output">
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabLog">
       <attribute name="title">
        <string>Log</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QTextBrowser" name="logBrowser"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabError">
       <attribute name="title">
        <string>Error</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QTextBrowser" name="errorBrowser"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="minimumSize">
           <size>
            <width>384</width>
            <height>512</height>
           </size>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listModel">
          <property name="minimumSize">
           <size>
            <width>192</width>
            <height>512</height>
           </size>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::MultiSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="alreadyConnected">
      <property name="text">
       <string>Write model already converter in file [AlreadyConverted.csv] ( if exists )</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
